### PR TITLE
Add `rows` icon

### PR DESCRIPTION
**Pull request by danielbayley**
https://lucide.dev/icons?search=columns

Related to #1003.

### CODE REVIEW

File: icons/rows.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "layout"
+  ],
+  "categories": [
+    "text"
```
Comment (karsa-mistmere) [inline icons/rows.json:7]:
```suggestion
    "text",
    "layout"
```

File: icons/rows.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "layout"
```
Comment (karsa-mistmere) [inline icons/rows.json:4]:
```suggestion
    "layout",
    "lines",
    "queue",
    "series",
    "list",
    "vertical",
    "horizontal"
```
(vertically listed items that have a horizontal orientation that's why both)

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]
Yeah, this is a no brainer, although I think both icons should also have the `layout` category, and definitely some extra tags.

**Comment by danielbayley** [comment]
> Icon

Moved to #1003…

**Review by jguddas — APPROVED** [review]

**Review by karsa-mistmere — APPROVED** [review]
LGTM!

**Comment by karsa-mistmere** [comment]
This was closed automatically because of #1003

**Review by ericfennis — APPROVED** [review]

**Pull request merged**